src/factories/calorimetry/CalorimeterHitsMerger_factory.h

```diff
@@ -22,6 +22,7 @@ class CalorimeterHitsMerger_factory : public JOmniFactory<CalorimeterHitsMerger_
     ParameterRef<std::string> m_readout {this, "readout", config().readout};
     ParameterRef<std::vector<std::string>> m_fields {this, "fields", config().fields};
     ParameterRef<std::vector<int>> m_refs {this, "refs", config().refs};
+    ParameterRef<std::vector<std::string>> m_mappings {this, "mappings", config().mappings};
```

Comment:
Mappings sounds a bit generic to me, and it doesn't self-describe that there should be as many mappings as fields. Should we call it something like field transformations or rewrites?
```suggestion
    ParameterRef<std::vector<std::string>> m_field_transformations {this, "fieldTransformations", config().field_transformations};
```

Reply:
Good point! I also like that name!